repos/system_upgrade/common/actors/systemd/checksystemdbrokensymlinks/actor.py

```diff
@@ -0,0 +1,23 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import checksystemdbrokensymlinks
+from leapp.models import SystemdBrokenSymlinksSource
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckSystemdBrokenSymlinks(Actor):
+    """
+    Detects and reports broken systemd symlinks
+
+    Two different reports are created - symlinks which have the same filename
+    as an existing and enabled systemd service are reported in
```

Comment:
It is still early Monday but this paragraph does not make sense to me. Could you please try to rephrase it?

Reply:
I think it's more understandable now.